### ISSUE TITLE
Cannot do a full clone ...

### ISSUE THREAD

**Issue by g40** [arteme/pod-ui#38]
Not quite sure of the problem here. Any thoughts?

Using ssh:
```
$ git clone --recurse-submodules git@github.com:arteme/pod-ui.git
Cloning into 'pod-ui'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
And then trying via HTTPS, gets stuck when cloning the sub=modules.

```
~/src
$ git clone --recurse-submodules https://github.com/arteme/pod-ui.git
Cloning into 'pod-ui'...
remote: Enumerating objects: 3724, done.
remote: Counting objects: 100% (703/703), done.
remote: Compressing objects: 100% (284/284), done.
remote: Total 3724 (delta 406), reused 687 (delta 400), pack-reused 3021
Receiving objects: 100% (3724/3724), 5.83 MiB | 7.38 MiB/s, done.
Resolving deltas: 100% (2367/2367), done.
Submodule 'build/resources' (git@github.com:arteme/pod-ui-resources) registered for path 'build/resources'
Cloning into '/home/moi/src/pod-ui/build/resources'...
The authenticity of host 'github.com (140.82.121.3)' can't be established.
ED25519 key fingerprint is xxx.
This key is not known by any other names.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'github.com' (ED25519) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:arteme/pod-ui-resources' into submodule path '/home/moi/src/pod-ui/build/resources' failed
Failed to clone 'build/resources'. Retry scheduled
Cloning into '/home/moi/src/pod-ui/build/resources'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:arteme/pod-ui-resources' into submodule path '/home/moi/src/pod-ui/build/resources' failed
Failed to clone 'build/resources' a second time, aborting

```

**Comment by arteme** [comment]
Hi,

 I'm not sure what's going on with the SSH connection. It's a silly thing to ask, but do you have a valid public key? For example, do you get the same:
```
 $ ssh -T git@github.com
Hi arteme! You've successfully authenticated, but GitHub does not provide shell access.
```

Since git sub-modules were using `git@github.com:...` URLs, the HTTPS clone didn't work, but I've replaced sub-modules URLs with relative URLs now, so `git clone --recurse-submodules https://github.com/arteme/pod-ui.git` should work correctly now.

**Comment by g40** [comment]
Hello @arteme, 

Thanks for the quick reply - yes, I've got a valid key but it looks like the msys environment is not quite configured correctly. 

Irrespective the HTTPS clone worked fine. After a bit of a scuffle with the tooling I have got a running build from source. A good result. Thanks again.

* might be worth noting in the readme/build notes for occasional msys users, like me, that they need the blue Msys mingw x64 shell. 

![image](https://github.com/arteme/pod-ui/assets/812250/39a45c1c-3a0e-4745-ae50-04f1fd0300db)
